### ISSUE TITLE
What are the plans for Funcotator?

### ISSUE THREAD

**Issue by pieterlukasse** [broadinstitute/gatk#8154]
## Bug Report

Not a bug, but a question. See **Description section** below.

### Affected tool(s) or class(es)
Funcotator

### Affected version(s)
- 4

### Description 

Looking around at https://github.com/broadinstitute/gatk/issues?q=label%3AFuncotator+is%3Aclosed I noticed not much happened since April 2022. Is there no more active development/maintenance planned for Funcotator? Or are the labels misleading? I am planning on writing a generic parser for Funcotator output, but would like to know a bit more about this project's status before investing in this. Thanks.

#### Steps to reproduce
See https://github.com/broadinstitute/gatk/issues?q=label%3AFuncotator+is%3Aclosed

**Comment by lbergelson** [comment]
Hi @pieterlukasse.  This is a great question and somewhat timely.  Funcotator hasn't gotten the attention it needs lately because the engineer who's most involved has been extremely busy with other very high priority projects.   We intend to support it going forward but it's unclear if that means bug fixes and reactive support or if we're able to make major upgrades.  We're currently in the middle of somewhat of a resource crunch, and we are actively planning how to prioritize our attention in the future.  

I think if you're basically happy with features now, you should feel safe investing the time into an output parser.  If there are major improvements you need I would wait a week or two and ask again then because we'll have more clarity about what we can do.  

If you're interested I would consider contributing back your code the GATK core.  There's some existing utility code that could probably help you, but there's a long standing gap in our tooling for users to make sense of the funcotations and we'd welcome improvements or new tools.

@jonn-smith @droazen

**Comment by jonn-smith** [comment]
@pieterlukasse Yeah - I'm planning on updating some of the Funcotator core to be more permissive for input data types and to fix a few long-standing bugs, but have been unable to do so because of other high-priority tasks (as @lbergelson said).

The output formats are pretty well-established, so I don't think there's any risk in writing an additional parser.  However if you simply want to view the outputs, you can render the annotations in `MAF` format and that will produce a `MAF` (TSV) file that is much more easily viewed / parsed.

**Comment by pieterlukasse** [comment]
@lbergelson and @jonn-smith thanks for your replies, they are really helpful. 

I do have some additional questions related to information you shared in your replies: 
>  There's some existing utility code that could probably help you

Any pointers to these parts are appreciated! 

> you can render the annotations in MAF format 

@jonn-smith is there also an option to take in an already annotated VCF and produce the corresponding MAF without having to annotate things again (so a simple conversion)? This is what I'm looking for at the moment.
